src/js/slider.js

```diff
@@ -0,0 +1,27 @@
+export default class Slider
+{
+  
+  constructor()
+  {
+    this.slideIndex = 1;
+    this.showSlides(this.slideIndex);
+  }
+  plusSlides(n) 
+  {
+    this.showSlides(this.slideIndex += n);
+  }
+  
+  showSlides(n) 
+  {
+    let i;
+    let slides = document.getElementsByClassName("image-slider_image");
+    if (n > slides.length) {this.slideIndex = 1}    
```

Comment:
Try to avoid using one-line code and format it as example below:
```suggestion
    if (n > slides.length) {
      this.slideIndex = STEP;
    }    
```